src/sentry/rules/actions/base.py

```diff
@@ -5,11 +5,28 @@
 from collections.abc import Generator
 
 from sentry.eventstore.models import GroupEvent
+from sentry.models.rule import Rule
 from sentry.rules.base import CallbackFuture, EventState, RuleBase
 
 logger = logging.getLogger("sentry.rules")
 
 
+def instantiate_action(rule: Rule, action):
```

Comment:
did we decide we watned this here in the `base.py` file? Or create a separate helpers/utils file?